cli/src/cmd/forge/test/mod.rs

```diff
@@ -466,6 +466,21 @@ fn short_test_result(name: &str, result: &TestResult) {
     println!("{status} {name} {}", result.kind.report());
 }
 
+/**
+ * Formats the aggregated summary of all test suites into a string (for printing)
+ */
+fn format_aggregated_summary(
+    num_test_suites: usize,
+    total_passed: usize,
+    total_failed: usize,
+    total_skipped: usize,
+) -> String {
+    format!(
+        "Test suites: {}. Total passed: {}. Total failed: {}. Total skipped: {}",
```

Comment:
Instead of 
```
Test suites: 4. Total passed: 33. Total failed: 7. Total skipped: 5
```

Wonder what people think of:

```
Ran 4 test suites: 33 tests passed, 7 failed, 5 skipped (45 total tests)
```

Reply:
Maybe something like this would be better for the user to view
```
--------------------------------------------------------------------------------
📋 Test Suites: 4
--------------------------------------------------------------------------------
✅ Passed: 33
❌ Failed: 7
⚠️ Skipped: 5
--------------------------------------------------------------------------------
```

Am not sure how the emojis will be displayed with different encoding standards

@mds1 @Evalir

Reply:
I think i'd rather keep it simple—I like @mds1 's suggestion!